src/helpers/versioning.ts

```diff
@@ -16,7 +16,13 @@ export const checkThemeStatus = async (state: vscode.Memento) => {
     const packageVersion = getCurrentExtensionVersion();
 
     // check if the theme was used before
-    if (stateVersion === undefined) {
+    if (
+      stateVersion === undefined ||
+      !(
```

Comment:
## Add type guard

Argument of type 'unknown' is not assignable to parameter of type
'string | SemVer'
ts(2345)

Probleme:
const stateVersion: unknown

Result:
const stateVersion: string | semver.SemVer

Reply:
Thanks! These changes are also very good!